itests/pom.xml

```diff
@@ -17,6 +17,7 @@
   <name>openHAB Add-ons :: Integration Tests</name>
 
   <modules>
+    <module>org.openhab.persistence.mapdb.tests</module>    
```

Comment:
Why is this the only integration test here?

Reply:
Because we don't have any other - they didn't exist in openHAB 1 and that's where all (but this) persistence services came from.

Reply:
Note that you are looking at the master branch here, not at 2.5.x!

Reply:
I knew I was looking at master, but I didn't know that master got rid of all the integration tests!

Reply:
It got rid off ALL the add-ons (until the moment that we do a bulk move from the 2.5.x branch to master later this year).